mbed_lstools/windows.py

```diff
@@ -38,22 +36,25 @@
 
 
 MAX_COMPOSITE_DEVICE_SUBDEVICES = 5
-MBED_STORAGE_DEVICE_VENDOR_STRINGS = ['ven_mbed', 'ven_segger', 'ven_arm_v2m', 'ven_nxp']
+MBED_STORAGE_DEVICE_VENDOR_STRINGS = [
+    "ven_mbed", "ven_segger", "ven_arm_v2m", "ven_nxp"
+]
 
 
 def _get_values_with_numeric_keys(reg_key):
     result = []
     try:
         for v in _iter_vals(reg_key):
+
```

Comment:
When do we actually put extra lines in for `for` loops? This doesn't appear to be consistent within this file.

Ex. https://github.com/ARMmbed/mbed-ls/pull/342/files#diff-49012471517cafe043a3614156f872e7R226